### PR TITLE
Added must-watch CSS

### PR DESCRIPTION
**Pull request by AllThingsSmitty**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
 ## Front-end Development
 
+- [CSS Must Watch Talks](https://github.com/AllThingsSmitty/must-watch-css)
```
Comment (sindresorhus) [inline readme.md]: Should be added to the bottom.

**Comment by sindresorhus** [comment]
Some commentary on why each item is recommended would make the list a lot more useful. Right now it's just a list of random list. I have no idea **why** I should bother watching any of these.

**Comment by AllThingsSmitty** [comment]
Adding this list of many well-received talks about CSS, with topics including working with CSS frameworks, Sass, improving CSS performance, knowing SVG and CSS animation, scalable CSS, mobile tips, et al., for how to get better and know more about CSS.

**Comment by sindresorhus** [comment]
Not here. In your repo.

**Comment by sindresorhus** [comment]
Thanks. This looks good. Would be cool if you could link back here in your list so users can easily discover more awesome lists ;)

**Comment by AllThingsSmitty** [comment]
Happy to [oblige](https://github.com/AllThingsSmitty/must-watch-css/). :wink:

**Pull request closed without merging**